specification/machinelearningservices/data-plane/Microsoft.MachineLearningServices/preview/2019-05-01/execution.json

```diff
@@ -0,0 +1,1103 @@
+        }
+      }
+    },
+    "SparkSection": {
+      "type": "object",
+      "properties": {
+        "repositories": {
+          "description": "The list of spark repositories.",
+          "uniqueItems": false,
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "packages": {
+          "description": "The Spark packages to use.",
+          "uniqueItems": false,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SparkMavenPackage"
+          }
+        },
+        "precachePackages": {
+          "description": "Whether to precache the packages.",
+          "type": "boolean",
+          "example": "True"
+        }
+      }
+    },
+    "ContainerRegistry": {
+      "type": "object",
+      "properties": {
+        "address": {
+          "type": "string"
+        },
+        "username": {
+          "type": "string"
+        },
+        "password": {
+          "type": "string"
+        }
+      }
+    },
+    "SparkMavenPackage": {
+      "type": "object",
+      "properties": {
+        "group": {
+          "type": "string"
+        },
+        "artifact": {
+          "type": "string"
+        },
+        "version": {
+          "type": "string"
+        }
+      }
+    },
+    "ExperimentRunDetails": {
+      "description": "Contains the details of a run.",
+      "required": [
+        "runId",
+        "target"
+      ],
+      "type": "object",
+      "properties": {
+        "runId": {
+          "description": "The identifier for a run.",
+          "type": "string",
+          "example": "myexperiment_155000000001_0"
+        },
+        "target": {
+          "description": "Target refers to compute where the job is scheduled for execution.",
+          "type": "string",
+          "example": "local"
+        }
+      }
+    },
+    "ErrorResponse": {
+      "description": "The error response.",
+      "type": "object",
+      "properties": {
+        "error": {
+          "$ref": "#/definitions/RootError",
+          "description": "The top level error that occurred."
+        },
+        "correlation": {
+          "description": "Dictionary containing correlation details for the error.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "environment": {
+          "description": "The hosting environment.",
+          "type": "string"
+        },
+        "location": {
+          "description": "The Azure region.",
+          "type": "string"
+        },
+        "time": {
+          "format": "date-time",
+          "description": "The time in UTC.",
+          "type": "string"
+        }
+      }
+    },
+    "RootError": {
+      "description": "The root error.",
+      "type": "object",
+      "properties": {
+        "code": {
+          "description": "The service-defined error code.",
```

Comment:
Where are the actual code values documented? Providing a link here would be great!

Reply:
We dont have an public facing docs yet. I have updated the xml comment with expected code values

Reply:
done